src/Workspaces/CoreTest/WorkspaceTests/MSBuildWorkspaceTests.cs

```diff
@@ -2372,6 +2372,7 @@ public void TestProjectReferenceWithNoGuid()
             }
         }
 
+        [WorkItem(5668, "https://github.com/dotnet/roslyn/issues/5668")]
```

Comment:
Should we have a test that explicitly sets the culture to something else? Doesn't the compiler have tests like that? Otherwise we could regress this and us testing on en-US only won't catch it.